app/joinEvent/index.tsx

```diff
@@ -56,19 +57,31 @@ export default function EventCreation() {
   const router = useRouter();
 
   const onSubmit = () => {
-    refetch().then((response) => {
-      if (response.data?.docs.length === 0) {
-        alert("Event not found");
-      } else {
-        const eventId = response.data?.docs[0].id;
-        if (eventId !== undefined) {
-          router.push({
-            pathname: "/eventPage/[id]",
-            params: { id: eventId },
+    if (value !== "") {
+      refetch().then((response) => {
+        if (response.data?.docs.length === 0) {
+          showToastable({
+            message: "Whoops! Event not found",
+            duration: 2000,
+            status: "success",
           });
+        } else {
+          const eventId = response.data?.docs[0].id;
+          if (eventId !== undefined) {
+            router.push({
+              pathname: "/eventPage/[id]",
+              params: { id: eventId },
+            });
+          }
         }
-      }
-    });
+      });
+    } else {
+      showToastable({
+        message: "Please enter the code!",
+        duration: 2000,
+        status: "success",
+      });
+    }
```

Comment:
I think we can remove this. #33 is adding the disabled property to the join button so that you can't press it when you haven't filled out the code

Reply:
Oh I see, well I guess it doesn't hurt to have it, to be extra safe 😇 even tho in theory it should never happen you never know